docs/src/pages/demos/lists/AlignItemsList.tsx

```diff
@@ -68,6 +72,9 @@ function AlignItemsList(props: AlignItemListProps) {
           }
         />
       </ListItem>
+      <li>
```

Comment:
I would follow @mbrookes suggestion and add `role="divider"`

Reply:
Right, it should be `role="separator"`.
https://www.w3.org/TR/html52/grouping-content.html#the-li-element
https://a11y-guidelines.orange.com/web_EN/exemples/simple-menu/simple-menu.html

I'm moving the logic in the core.

Reply:
>> @mbrookes suggestion and add role="divider"

> it should be role="separator"

That. :sweat_smile:

Reply:
:)